sktime/utils/validation/__init__.py

```diff
@@ -12,7 +12,7 @@
     "check_n_jobs",
     "check_window_length",
 ]
-__author__ = ["Markus Löning", "Taiwo Owoseni", "khrapovs"]
+__author__ = ["mloning", "Taiwo Owoseni", "khrapovs"]
```

Comment:
what about `Taiwo Owoseni`?

Reply:
not caught by bulk-replace. Someone else can do that and do all the linting ;-)